<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #d7e7f5;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>641</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>800</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 italic 16pt &quot;Futura&quot;;
color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <item>
     <property name="text">
      <string>Inbox</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>    All Tasks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>    Today</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>    Tommorow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>    This Week</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>    Trash</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0060bf;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>50</y>
      <width>600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>800</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;</string>
    </property>
    <widget class="QLineEdit" name="add_task_qline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: white;
border: 1px dashed black;
border-width: 1px;
border-radius: 10px;</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>  Add a task...</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="task_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>591</width>
       <height>551</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-weight: normal;
font:  15pt &quot;Helvetica&quot;;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: white;
border-color: black;
border-style: dashed;
border-width: 1px;
border-radius: 10px;
padding: 2px;</string>
     </property>
     <property name="text">
      <string>Incomplete</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: white;
border-color: black;
border-style: dashed;
border-width: 1px;
border-radius: 10px;
padding: 2px;</string>
     </property>
     <property name="text">
      <string>Completed</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 6px;
border : 2px solid grey;
background-color: #dbe8f6;
color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="complete_main_checkbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>85</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>Complete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/ddf131967dc6b75a17a355a19ea22c61.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font: 22pt &quot;Helvetica&quot;;</string>
    </property>
    <property name="text">
     <string>All Tasks </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="alltasks_tasks" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #dbe8f6;</string>
    </property>
    <widget class="QLabel" name="tasks_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>tasks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tasks_num">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #0060bf;
font-size: 19px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="alltasks_overdue" native="true">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #dbe8f6;</string>
    </property>
    <widget class="QLabel" name="overdue_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>overdue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="overdue_num">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
font-size: 19px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="alltasks_completed" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #dbe8f6;</string>
    </property>
    <widget class="QLabel" name="completed_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>completed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="completed_num">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 19px;
color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
